assets/sass/components/global/_googlesitekit-dashboard-navigation.scss

```diff
@@ -0,0 +1,58 @@
+/**
+ * DashboardNavigation styles.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-navigation {
+
+		@include shadow;
+
+		background-color: $c-base;
+		justify-content: center;
+		left: 0;
+		padding: 12px 0;
+		position: sticky;
+		right: 0;
+		top: 68px;
+		z-index: 100001;
```

Comment:
@kuasha420 we also need to update the `.googlesitekit-header` class to have the same z-index value.

Reply:
Updated the value, hopefully it will not break anything else :grinning:

Reply:
Added a new ticket for it: #4360

Reply:
Done.